static/src/stylesheets/head.survey.scss

```diff
@@ -40,3 +40,9 @@
    ========================================================================== */
 
 @import 'module/atoms/_quiz';
+
+/* ==========================================================================
+   Survey
+   ========================================================================== */
+
+@import 'module/survey/survey';
```

Comment:
No more than we were already getting. Previously because of [this code](https://github.com/guardian/frontend/blob/master/common/app/assets/assets.scala#L116) we would get [head.content.scss](https://github.com/guardian/frontend/blob/master/static/src/stylesheets/head.content.scss) inlined, as well as [survey.scss](https://github.com/guardian/frontend/blob/master/static/src/stylesheets/survey.scss) loaded asynchronously. Since `head.survey` is just a copy of `head.content` plus the line above, we're not getting anything extra (in fact we're getting less because we're not getting [this stuff](https://github.com/guardian/frontend/blob/master/static/src/stylesheets/survey.scss#L1))

There is a question about how much of the stuff copied from `head.content` we actually need. We could try chopping out lines to see if anything's affected on the survey pages. I'd rather leave that for a separate PR though